src/http/api/views/account.follows.view.ts

```diff
@@ -37,6 +36,48 @@ interface AccountRow {
     avatar_url: string;
 }
 
+interface FollowsActor {
+    id: string;
+    name: string;
+    preferredUsername: string;
+    icon: {
+        url: string;
+    };
+}
+
+function isValidFollowsActor(obj: unknown): obj is FollowsActor {
```

Comment:
Kept this validation in a separate method as I had to do these checks at two places.